sample-linuxx64-app/src/linuxMain/kotlin/com/badoo/reaktive/sample/linux/MainScheduler.kt

```diff
@@ -0,0 +1,102 @@
+package com.badoo.reaktive.sample.linux
+
+import com.badoo.reaktive.disposable.CompositeDisposable
+import com.badoo.reaktive.scheduler.Scheduler
+import com.badoo.reaktive.scheduler.Scheduler.Executor
+import com.badoo.reaktive.utils.atomic.AtomicBoolean
+import com.badoo.reaktive.utils.atomic.AtomicList
+import com.badoo.reaktive.utils.atomic.atomicList
+import com.badoo.reaktive.utils.atomic.getAndUpdate
+import com.badoo.reaktive.utils.atomic.plusAssign
+import com.badoo.reaktive.utils.atomic.remove
+import kotlinx.cinterop.StableRef
+import kotlinx.cinterop.asStableRef
+import kotlinx.cinterop.reinterpret
+import kotlinx.cinterop.staticCFunction
+import libgtk3.FALSE
+import libgtk3.TRUE
+import libgtk3.g_source_remove_by_user_data
+import libgtk3.g_timeout_add
+import libgtk3.gpointer
+
+class MainScheduler : Scheduler {
```

Comment:
Do we want to expose this outside of the sample?

Reply:
You mean if someone will depend on this sample app?